src/backend/distributed/sql/citus--8.3-1--9.0-1.sql

```diff
@@ -170,4 +170,46 @@ CREATE FUNCTION worker_apply_sequence_command(create_sequence_command text,
 COMMENT ON FUNCTION worker_apply_sequence_command(text,regtype)
     IS 'create a sequence which produces globally unique values';
 
+CREATE TYPE isdatanode AS ENUM('true', 'marked for draining', 'false');
```

Comment:
isdatanode seems like a strange name for a type

maybe 'marked for draining' -> 'draining'.

Reply:
Do we really need 3 states? It seems to complicate the code a lot. Draining could also be (isdatanode=false, placements exist).